### PR TITLE
Add traffic analysis

### PR DESCRIPTION
**Pull request by IsaackMwenda**
modified from IBTCI

### CODE REVIEW

File: configurations/kalobeyei_pipeline_config.json [hunk 1]
```diff
@@ -122,25 +119,64 @@
...
+      {"StartDate": "2020-02-20T16:30+03:00", "EndDate": "2020-02-20T18:30+03:00", "Label": "E01 Juba-live"},
+      {"StartDate": "2020-02-21T16:00+03:00", "EndDate": "2020-02-21T18:00+03:00", "Label": "E01 Kiswahili-live"},
+      {"StartDate": "2020-02-21T20:00+03:00", "EndDate": "2020-02-21T22:00+03:00", "Label": "E01 Kiswahili-repeat"},
+      {"StartDate": "2020-02-22T20:00+03:00", "EndDate": "2020-02-22T22:00+03:00", "Label": "E01 Juba-repeat"},
+
+      {"StartDate": "2021-02-26T19:30+03:00", "EndDate": "2020-02-27T16:00+03:00", "Label": "E02 Ad"},
+      {"StartDate": "2020-02-27T16:00+03:00", "EndDate": "2020-02-27T18:00+03:00", "Label": "E02 Juba-live"},
+      {"StartDate": "2020-02-28T16:00+03:00", "EndDate": "2020-02-28T18:00+03:00", "Label": "E02 Kiswahili-live"},
+      {"StartDate": "2020-02-28T20:00+03:00", "EndDate": "2020-02-28T22:00+03:00", "Label": "E02 Kiswahili-repeat"},
+      {"StartDate": "2020-02-22T20:00+03:00", "EndDate": "2020-02-22T22:00+03:00", "Label": "E02 Juba-repeat"},
```
Comment (as2388) [inline configurations/kalobeyei_pipeline_config.json:153]: This date looks suspect

File: configurations/kalobeyei_pipeline_config.json [hunk 2]
```diff
@@ -122,25 +119,64 @@
...
   ],
 
   "PipelineName": "leap_kalobeyei_s01_pipeline",
   "ProjectStartDate": "2020-01-18T13:00:00+03:00",
   "ProjectEndDate": "2100-01-01T00:00:00+03:00",
   "FilterTestMessages": true,
   "MoveWSMessages": true,
+  "AutomatedAnalysis": {
+    "TrafficLabels": [
+      {"StartDate": "2021-02-19T19:30+03:00", "EndDate": "2020-02-20T16:30+03:00", "Label": "E01 Ad"},
```
Comment (as2388) [inline configurations/kalobeyei_pipeline_config.json:143]: Does RDA agree with this definition? It's quite a long time for an advert

**Review by as2388 — APPROVED** [review]

File: configurations/kalobeyei_pipeline_config.json (hunk 1 above)
Comment (IsaackMwenda) [reply configurations/kalobeyei_pipeline_config.json]: Thanks for catching this. Updated.

File: configurations/kalobeyei_pipeline_config.json (hunk 2 above)
Comment (IsaackMwenda) [reply configurations/kalobeyei_pipeline_config.json:144]: Yes, this is okay for the camp context and based on past experiences. Promos have a negligible impact on engagement but we have people responding to the advert even after 12 hour period.

**Pull request merged**